### PR TITLE
Update patterns.ini

### PR DESCRIPTION
**Pull request by vxsh4d0w**

### CODE REVIEW

File: iocp/data/patterns.ini [hunk 1]
```diff
@@ -7,19 +7,19 @@ pattern:	\b(([a-z0-9\-]{2,}\[?\.\]?)+(abogado|ac|academy|accountants|active|acto
 defang:		True
 
 [IP]
-pattern:	\b(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\b
+pattern:	\b((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$\b
```
Comment (garanews) [inline iocp/data/patterns.ini]: ok but need to remove the $ at the end in order to match IP in any part of the document and not only at the end of the line (example: 8.8.8.8|google.com)

File: iocp/data/patterns.ini [hunk 2]
```diff
@@ -7,19 +7,19 @@ pattern:	\b(([a-z0-9\-]{2,}\[?\.\]?)+(abogado|ac|academy|accountants|active|acto
...
 
 [Email]
-pattern:	\b([a-z][_a-z0-9-.]+@[a-z0-9-]+\.[a-z]+)\b
+pattern:	\b([a-zA-Z0-9_\-\.]+)@([a-zA-Z0-9_\-\.]+)\.([a-zA-Z]{2,5})$\b
```
Comment (garanews) [inline iocp/data/patterns.ini]: Good idea, maybe it's better to use the W3C Email Regex used in HTML5 email field: 
\b[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)\b

File: iocp/data/patterns.ini [hunk 3]
```diff
@@ -7,19 +7,19 @@ pattern:	\b(([a-z0-9\-]{2,}\[?\.\]?)+(abogado|ac|academy|accountants|active|acto
...
 
 [MD5]
-pattern:	\b([a-f0-9]{32}|[A-F0-9]{32})\b
+pattern:	\b[a-fA-F0-9]{32}(:.+)?$\b
```
Comment (garanews) [inline iocp/data/patterns.ini]: ok but need to remove the $ at the end in order to match the MD5 in any part of the document and not only at the end of the line (example: a04e72503528dfc132c48e95fa3160ad|putty.exe)

File: iocp/data/patterns.ini [hunk 4]
```diff
@@ -7,19 +7,19 @@ pattern:	\b(([a-z0-9\-]{2,}\[?\.\]?)+(abogado|ac|academy|accountants|active|acto
...
 
 [SHA1]
-pattern:	\b([a-f0-9]{40}|[A-F0-9]{40})\b
+pattern:	\b[a-fA-F0-9]{40}(:.+)?$\b
```
Comment (garanews) [inline iocp/data/patterns.ini]: ok but need to remove the $ at the end in order to match the SHA1 in any part of the document and not only at the end of the line (example: 3be0cb487da1fffe05b9d1a3a39c9db77d5615cb|putty.exe)

File: iocp/data/patterns.ini [hunk 5]
```diff
@@ -7,19 +7,19 @@ pattern:	\b(([a-z0-9\-]{2,}\[?\.\]?)+(abogado|ac|academy|accountants|active|acto
...
 
 [SHA256]
-pattern:	\b([a-f0-9]{64}|[A-F0-9]{64})\b
+pattern:	\b[a-fA-F0-9]{64}(:.+)?$\b
```
Comment (garanews) [inline iocp/data/patterns.ini]: ok but need to remove the $ at the end in order to match the SHA256 in any part of the document and not only at the end of the line (example: 567efd7abb99428737d22bf3f8cee9d23a540b4e1565938b557eec54078b8a31|putty.exe)

File: iocp/data/patterns.ini (hunk 1 above)
Comment (vxsh4d0w) [reply iocp/data/patterns.ini]: Oki doki.

File: iocp/data/patterns.ini (hunk 5 above)
Comment (vxsh4d0w) [reply iocp/data/patterns.ini]: oki doki

File: iocp/data/patterns.ini (hunk 4 above)
Comment (vxsh4d0w) [reply iocp/data/patterns.ini]: oki doki

File: iocp/data/patterns.ini (hunk 3 above)
Comment (vxsh4d0w) [reply iocp/data/patterns.ini]: oki doki

File: iocp/data/patterns.ini (hunk 2 above)
Comment (vxsh4d0w) [reply iocp/data/patterns.ini]: oki doki

**Review by vxsh4d0w — COMMENTED** [review]
These changes fix last review made a month ago.

**Comment by vxsh4d0w** [comment]
This update fix last review made a month ago

**Pull request closed without merging**